src/schematic/schematic_text.ts

```diff
@@ -25,9 +26,10 @@ export const schematic_text = z.object({
   schematic_component_id: z.string().optional(),
   schematic_text_id: z.string(),
   text: z.string(),
+  font_size: z.number().default(1),
   position: z.object({
-    x: distance,
-    y: distance,
+    x: distance.default(0),
+    y: distance.default(0),
```

Comment:
i don't think this should have a default value, default values with position cause a lot of problems imo because you can't catch errors early. font_size is ok however